packages/bpk-component-theme-toggle/src/updateOnThemeChange-test.js

```diff
@@ -17,21 +17,34 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import renderer from 'react-test-renderer';
 import { mount } from 'enzyme';
 import updateOnThemeChange from './updateOnThemeChange';
 import { THEME_CHANGE_EVENT, getHtmlElement } from './utils';
 
-const EnhancedComponent = updateOnThemeChange('div');
+const Dummy = ({ children }) => <div>{children}</div>;
+
+const EnhancedComponent = updateOnThemeChange(Dummy);
```

Comment:
The enhanced component has the PropType `children` required, so I've added a dummy component to comply with the requirement